config/routes.rb

```diff
@@ -11,6 +11,11 @@
     end
     resources :comments, only: :create
   end
+  resources :users do
+    member do
+      post :follow
```

Comment:
All of that will be added in another pull request, I just wanted this merged because I know other people have been merging stuff and I wanted this added before any new merges, to avoid a merge conflict.